webextensions/browser-compat-data.json

```diff
@@ -1212,19 +1215,24 @@
       }, 
       "Firefox": {
         "notes": [
-          "Only the 'basic' type is supported."
+          "Only the 'basic' type is supported.",
+          "For the 'basic' type, 'iconUrl' is optional."
         ], 
         "support": "45.0"
       }, 
       "Firefox for Android": {
         "notes": [
-          "Only the 'basic' type is supported."
+          "Only the 'basic' type is supported.",
+          "For the 'basic' type, 'iconUrl' is optional."
         ], 
         "support": "48.0"
       }
     }, 
     "create": {
       "Chrome": {
+        "notes": [
+          "For the 'basic' type, if 'iconUrl' is present, but the URL can not be loaded, the notification will not be displayed, an error will be shown in the extension's background page console and 'runtime.lastError' will be set in the callback function."
+        ],
```

Comment:
I think the disconnect between you seeing this as just restating what was in [NotificationOptions](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/notifications/NotificationOptions) and my intent of clarification was that the NotificationOptions page was erroneously stating that iconUrl is mandatory in Firefox when it is actually optional in Firefox, but not in Chrome.